### PR TITLE
#164798248 Setup SASS processing on the client

### PR DESCRIPTION
**Pull request by e-ian**
#### What does this PR do?
- Set up sass processing on the client.
#### Description of tasks to be completed.
- Install css loader, sass-loader and node-sass
- Set up sass pre-processing by configuring it in webpack.config.js
- Set indentation to two spaces
#### How should this be manually tested?
- Clone this repo :
   `git clone ah-frontend-prime`

- Git checkout to this branch :
   `git checkout ch-setup-sass-processing-164798248`

- Install all dependencies :
   `npm install `

- Run application:
   `npm start`
#### Any background context you want to provide?
- N/A

#### What are the relevant pivotal tracker stories?
[#164798248](https://www.pivotaltracker.com/story/show/164798248)
#### Screenshots (if appropriate)
image

### CODE REVIEW

File: src/index.scss [hunk 1]
```diff
@@ -0,0 +1,5 @@
+body {
+    div#root{
+      background-color: rgb(255, 255, 255);
+    }
+}
```
Comment (patrickf949) [inline src/index.scss]: @e-ian great work 
please add a new line at the end of this file though

**Review by anyatibrian — APPROVED** [review]
LGTM

File: src/index.scss (hunk 1 above)
Comment (e-ian) [reply src/index.scss]: Done @patrickf949

**Review by patrickf949 — APPROVED** [review]
LGTM

**Review by Genza999 — APPROVED** [review]
Great work @e-ian , LGTM!

**Review by MuhanguziDavid — COMMENTED** [review]
Nice work @e-ian , please follow the engineering playbook conventions in your commit message and PR description e.g.
- the description should be richer with sections like `How should this be manually tested?`
- the points in the commit subject and commit body should not start with an upper case letter
[Engineering playbook conventions](https://github.com/andela/engineering-playbook/tree/e654a2dd79972dfe51a1b54d6c7ebc04d2c5e012/5.%20Developing/Conventions#branch-naming)

**Comment by e-ian** [comment]
> Nice work @e-ian , please follow the engineering playbook conventions in your commit message and PR description e.g.
> 
> * the description should be richer with sections like `How should this be manually tested?`
> * the points in the commit subject and commit body should not start with an upper case letter
>   [Engineering playbook conventions](https://github.com/andela/engineering-playbook/tree/e654a2dd79972dfe51a1b54d6c7ebc04d2c5e012/5.%20Developing/Conventions#branch-naming)

Hello @MuhanguziDavid. I have implemented the feedback as recommended

File: src/index.scss [hunk 2]
```diff
@@ -0,0 +1,5 @@
+body {
+    div#root{
```
Comment (MuhanguziDavid) [inline src/index.scss]: @e-ian , please use indentation of two spaces here, and everywhere else in the app.

Comment (e-ian) [reply src/index.scss]: I have set indentation to two spaces in all files

**Review by MuhanguziDavid — APPROVED** [review]
LGTM

**Pull request merged**